zk_toolbox/crates/zk_inception/src/commands/chain/deploy_paymaster.rs

```diff
@@ -51,8 +55,10 @@ pub async fn deploy_paymaster(
     forge.run(shell)?;
     spinner.finish();
 
-    let output =
-        DeployPaymasterOutput::read(shell, DEPLOY_PAYMASTER.output(&chain_config.link_to_code))?;
+    let output = DeployPaymasterOutput::read(
+        shell,
+        DEPLOY_PAYMASTER_SCRIPT_PARAMS.output(&chain_config.link_to_code),
+    )?;
```

Comment:
could we use read with a base path?

Reply:
same question for all forge interactions

Reply:
e.g. You can have path in the trait that starts from `repo path` and you will provide as a base path link to code

Reply:
This could be done with the new trait `ReadConfigWithBasePath`. For this to work, we'd need to modify `ForgeScriptParams` to have `output` field as `pub`, in order to have the following implementation:

```rust
impl FileConfigWithDefaultName for DeployPaymasterOutput {
    const FILE_NAME: &'static str = DEPLOY_PAYMASTER_SCRIPT_PARAMS.output;
}
```

Also, we'd need to modify the output to have the inner path to `contracts/l1-contracts-foundry`:

```rust
pub const DEPLOY_PAYMASTER_SCRIPT_PARAMS: ForgeScriptParams = ForgeScriptParams {
    output: "contracts/l1-contracts-foundry/script-out/output-deploy-paymaster.toml",
    //...
};
```

Now this can be used like this:

```rust
let output = DeployPaymasterOutput::read_with_base_path(shell, chain_config.link_to_code)?;
```

A few trade-offs, but seems to work. If you're ok with the implementation I can do it.

Reply:
That's a good suggestion, but we can probably make a step further and introduce some helpers to work with forge. Now it's a bit of complex